app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const fs = require('fs')
+const {
+    getTopics
+} = require('./db/controller/topic-controller');
+
+app.use(express.json());
+app.get('/api/topics', getTopics);
+app.get('/api', (request, response) => {
+    const endpoints = JSON.parse(fs.readFileSync('./endpoints.json', 'utf8'));
```

Comment:
Although reading a file synchronously works here, it's better to use the asynchronous version to avoid blocking other operations while the file is being read.